### PR TITLE
Remove other language examples and support

### PR DESCRIPTION
**Pull request by tjs-intel**

### CODE REVIEW

File: web/js/component/code_block.js [hunk 1]
```diff
@@ -23,13 +23,10 @@ import {isDefinedAndNotNull, replaceAll} from "../utils";
...
 const LANGUAGE_PROP_TYPE = PropTypes.oneOf([
   "java",
-  "r",
   "python",
   "bash",
   "json",
-  "matlab",
   "yaml",
-  "dockerfile",
   "plaintext",
```
Comment (Chris-Sigopt) [inline web/js/component/code_block.js:30]: Why dockerfile? Is that for orchestrate?

**Review by Chris-Sigopt — APPROVED** [review]
LGTM.

File: web/js/component/code_block.js (hunk 1 above)
Comment (tjs-intel) [reply web/js/component/code_block.js:30]: We had dockerfile examples for orchestrate back when we hosted the orchestrate docs at app.sigopt.com/docs

Comment (Chris-Sigopt) [reply web/js/component/code_block.js:30]: If that's already gone then great.

File: web/js/pages/experiment/common/api/page.js [hunk 2]
```diff
@@ -46,11 +45,6 @@ const CORE_MODULE_CODE_MAP = {
       <PythonExperimentCodeGen {...props} />
     </CopyableText>
   ),
```
Comment (tjs-intel) [inline web/js/pages/experiment/common/api/page.js:47]: It will be a little awkward having only 1 option in the language selection but 🤷

**Pull request merged**